corehq/apps/importer/views.py

```diff
@@ -109,6 +121,44 @@ def excel_config(request, domain):
 @require_POST
 @require_can_edit_data
 def excel_fields(request, domain):
+    """
+    Step one of three.
+
+    Important values that are grabbed from the POST or defined by
+    the user on this page:
+
+    named_columns:
+        Passed through from last step, see that for documentation
+
+    case_type:
+        The type of case we are matching to. When creating new cases,
+        this is the type they will be created as. When updating
+        existing cases, this is the type that we will search for.
+        If the wrong case type is used when looking up existing cases,
+        we will not up date them.
+
+    search_field:
+
+    create_new_cases:
+        A boolean that controls whether or not the user wanted
+        to create new cases for any case that doesn't have a matching
+        case id in the upload.
+
+    search_column:
+        Which column of the excel file we are using to specify either
+        case ids or external ids. This is, strangely, required. If
+        creating new cases only you would expect these to be blank with
+        the create_new_cases flag set.
+
+    search_field:
+        Either case id or external id, determines which type of
+        identification we are using to match to cases.
+
+    key_column/value_column:
+        These correspond to an advanced feature allowing a user
+        to modify a single case with multiple rows.
```

Comment:
might be a good idea to point to an upload script.  I think we've got one in our repo somewhere, don't we?  If not, I wrote one for TDH we could put in as documentation.